features/rhelah/atomic/rpm-ostree_bad_ca_cert.feature

```diff
@@ -0,0 +1,16 @@
+Feature: tests whether RH CA pinning works for rpm-ostree
+    It is critical for security that we're "pinned" to the RHN CA cert. If another server certificate is presented we should error out with something like Unacceptable TLS certificate.
+
+Scenario: 0. Host provisioned and subscribed
+    When "all" host is auto-subscribed to "prod"
+    Then subscription status is ok on "all"
+    and "1" entitlement is consumed on "all"
+
+Scenario: 1. Prepare mitmproxy
+    Given "git" are already installed on "rhel7"
+    and checkout the git repo "https://github.com/ljozsa/tools.git" on "rhel7"
+    and run command "command /root/tools/mitmproxy_install.sh" on "rhel7"
+
+Scenario: 2. Direct all https traffic through mitmproxy.
+    When IP address of the "rhel7" is recorded
```

Comment:
I'm not sure this step is necessary since the following step performs the `get_ipaddr()` on the same host.